explorer/src/components/account/VotesCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { VoteAccount, Vote } from "validators/accounts/vote";
+
+export function VotesCard({ voteAccount }: { voteAccount: VoteAccount }) {
+  return (
+    <>
+      <div className="card">
+        <div className="card-header">
+          <div className="row align-items-center">
+            <div className="col">
+              <h3 className="card-header-title">Votes History</h3>
+            </div>
+          </div>
+        </div>
+
+        <div className="table-responsive mb-0">
+          <table className="table table-sm table-nowrap card-table">
+            <thead>
+              <tr>
+                <th className="w-1 text-muted">Slot</th>
+                <th className="text-muted">Confirmation Count</th>
+              </tr>
+            </thead>
+            <tbody className="list">
+              {voteAccount.info.votes.length > 0 &&
+                voteAccount.info.votes.map((vote: Vote, index) =>
```

Comment:
nit: we display slots in descending order in most places, I think it would be nice to reverse the array here